stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -1122,28 +1122,6 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
         _sliceRange = 0..<count
     }
     
-    /// Initialize a `Data` with the contents of an Array.
-    ///
-    /// - parameter bytes: An array of bytes to copy.
-    public init(bytes: Array<UInt8>) {
```

Comment:
The Data perf tests have some cases that cover these specific types, right?

Reply:
Yes it does